### PR TITLE
docs(exploring-data): Add upload excel notes

### PR DESCRIPTION
**Pull request by aniaan**
### SUMMARY

Add upload excel notes

### ADDITIONAL INFORMATION

- [x] Has associated issue: #17494 
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

cc: @zhaoyongjie

### CODE REVIEW

File: docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx [hunk 1]
```diff
@@ -26,6 +26,9 @@ following information about each flight is given:
 You may need to enable the functionality to upload a CSV or Excel file to your database. The following section
 explains how to enable this functionality for the examples database.
 
+**Note:** To enable `Upload Excel`, you need to make sure that the `xlrd` dependency is installed,
+which you can execute `pip install apache-superset[excel]` to install.
+
```
Comment (villebro) [inline docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx:31]: While you're at it, would you mind updating line 26 to state that we also support The Parquet columnar format for data uploads?

Comment (aniaan) [reply docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx:31]: @villebro  Of course, it has been updated.

File: docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx [hunk 2]
```diff
@@ -23,9 +23,12 @@ following information about each flight is given:
 
 ### Enabling Data Upload Functionality
 
-You may need to enable the functionality to upload a CSV or Excel file to your database. The following section
+You may need to enable the functionality to upload a CSV, Excel file, Columnar file to your database. The following section
```
Comment (villebro) [inline docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx:26]: nit: I would change the comma to an "or", but I'm mostly indifferent with "Columnar" vs "Parquet" (for now we only support Parquet, but in the future we may add support for other columnar formats).
```suggestion
You may need to enable the functionality to upload a CSV, Excel file or Parquet file to your database. The following section
```

Comment (aniaan) [reply docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx]: @villebro it has been updated.

**Review by villebro — APPROVED** [review]
Thanks @aniaan ! Merging when CI is green

**Pull request merged**